### ISSUE TITLE
Plain zero accuracy

### ISSUE THREAD

**Issue by Sebagam** [Project-MONAI/tutorials#122]
Hi, I'm getting lowering loss scores across epochs, but accuracy is always zero.

Here is my google colab code:

https://colab.research.google.com/drive/1XcHHnNvxVCbTDV7bgnWHrSjGD7XKzY_f?usp=sharing

Can you tell me where the issue is?

Thanks

**Comment by Nic-Ma** [comment]
Hi @ericspod ,

Could you please help check this training issue? I am you are expert on these research topics.
Thanks in advance.

**Comment by rijobro** [comment]
It looks like your training set uses the first 10 images (presumably from the AD folder), and your validation uses the last 10 images (presumably from DFT). 

I think you're missing a shuffle of the files: `random.shuffle(images)` otherwise you're only training your network with one class, and validating with the other.

Perhaps you also wanted to use `[:-10]` for the training rather than `[:10]` to use all data except for those used for validation?

**Comment by Sebagam** [comment]
Thanks! Now it's working.

**Comment by rijobro** [comment]
No problem. For future reference, these sort of questions are better asked under the [Discussions tab](https://github.com/Project-MONAI/MONAI/discussions) of MONAI's main repository.
